docs/index.rst

```diff
@@ -15,17 +15,11 @@ Here's normal usage of the server:
 .. code-block:: python
 
    from waitress import serve
-   serve(wsgiapp, host='0.0.0.0', port=8080)
+   serve(wsgiapp, listen='*:8080')
 
-If you want to serve your application on all IP addresses, on port 8080, you
-can omit the ``host`` and ``port`` arguments and just call ``serve`` with the
-WSGI app as a single argument:
+This will run waitress on port 8080 on all available IP addresses, both IPv4
+and IPv6.
 
-.. code-block:: python
-
-   from waitress import serve
-   serve(wsgiapp)
-   
```

Comment:
That's a good idea. Will add it back in. Right now it continues to use host/port as the defaults in that case.